nixos/tests/mysql/mysql.nix

```diff
@@ -185,19 +185,5 @@ import ./../make-test-python.nix ({ pkgs, ...} : {
     mariadb.succeed(
         "echo 'use testdb; drop table rocksdb;' | sudo -u testuser mysql -u testuser"
     )
-  '' + pkgs.stdenv.lib.optionalString pkgs.stdenv.isx86_64 ''
```

Comment:
Why are you dropping this?

Reply:
In MariaDB 10.5 dropping support TokuDB plugin.

Reply:
Source? [This](https://mariadb.com/docs/features/whats-new/whats-new-in-community-server-105/#tokudb-storage-engine) just says they're not including it in packages anymore? Does that equal dropping support in MariaDB-speak?

Reply:
TokuDB building with this patch
```
diff --git a/storage/tokudb/CMakeLists.txt b/storage/tokudb/CMakeLists.txt
index bc7e5826..eaad6488 100644
--- a/storage/tokudb/CMakeLists.txt
+++ b/storage/tokudb/CMakeLists.txt
@@ -38,7 +38,6 @@ SET(TOKUDB_SOURCES
     tokudb_thread.cc
     tokudb_dir_cmd.cc)
 MYSQL_ADD_PLUGIN(tokudb ${TOKUDB_SOURCES} STORAGE_ENGINE MODULE_ONLY
-                 DISABLED
                  COMPONENT tokudb-engine CONFIG ${CMAKE_CURRENT_BINARY_DIR}/tokudb.cnf)

 IF(NOT TARGET tokudb)
```
Need return?

Reply:
If enabling it needs patching like this, I'd say disabling it is the way to go.
That just wasn't clear to me from the MariaDB documentation.